Extension/walkthrough/installcompiler/install-gcc-linux-short.md

```diff
@@ -0,0 +1,10 @@
+<h1 data-loc-id="walkthrough.linux.install.compiler">Install a C++ compiler on Linux</h1>
+<p>You can install the GCC compiler on Linux by running the following commands in a terminal:</p>
```

Comment:
Should we specify that these are instructions for Ubuntu and related distros?  Some other distros use `yum`.

Reply:
There are other distros as well, so this should probably refer to "or other package managers".  But, resolving this for now, to be addressed in a subsequent PR.